### DISCUSSION TITLE
Enhancement: Add benchmark(s)

### DISCUSSION THREAD

**Discussion by iron3oxide** [yezz123/ormdantic#45] [Ideas]
Even though performance isn't the main argument for using ormdantic, one might want to know how it fares in a typical scenario compared to other ORMs, e.g. sqlmodel, PonyORM, sqlalchemy etc. If the example app (see #39) is designed well, it could double as a benchmark app, meaning there is a slightly modified version in the repo (or another, designated benchmark repo) for each ORM.

**Comment by yezz123** [comment]
I'm planning to add benchmarks ASAP I start with this repo to set up and test several functions that ormdantic provides ex. create, update, delete, etc...

https://github.com/yezz123/TodoList-Ormdantic-starter
